diff --git a/node_modules/ra-core/esm/core/createAdminStore.js b/node_modules/ra-core/esm/core/createAdminStore.js
index ea45a3f..8fba27b 100644
--- a/node_modules/ra-core/esm/core/createAdminStore.js
+++ b/node_modules/ra-core/esm/core/createAdminStore.js
@@ -70,7 +70,7 @@ export default (function (_a) {
         });
     };
     var sagaMiddleware = createSagaMiddleware();
-    var typedWindow = window;
+    var typedWindow = typeof window !== 'undefined' && window;
     var composeEnhancers = (process.env.NODE_ENV === 'development' &&
         typeof typedWindow !== 'undefined' &&
         typedWindow.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ &&
