diff --git a/node_modules/ra-data-fakerest/esm/index.js b/node_modules/ra-data-fakerest/esm/index.js
index 9a95d24..281135c 100644
--- a/node_modules/ra-data-fakerest/esm/index.js
+++ b/node_modules/ra-data-fakerest/esm/index.js
@@ -46,7 +46,7 @@ export default (function (data, loggingEnabled) {
     if (loggingEnabled === void 0) { loggingEnabled = false; }
     var restServer = new FakeRest.Server();
     restServer.init(data);
-    if (window) {
+    if (typeof window !== 'undefined') {
         // give way to update data in the console
         window.restServer = restServer;
     }
